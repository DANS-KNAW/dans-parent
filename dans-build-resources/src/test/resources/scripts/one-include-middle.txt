One include here,

#include <multi-line.txt>

in the middle.