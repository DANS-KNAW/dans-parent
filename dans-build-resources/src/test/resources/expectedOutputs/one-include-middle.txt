One include here,

line 1
line 2
line 3


in the middle.